sdks/python/src/opik/integrations/langchain/opik_tracer.py

```diff
@@ -344,6 +349,65 @@ def _on_llm_start(self, run: "Run") -> None:
 
         self._process_start_span(run)
 
+    def on_chat_model_start(
+        self,
+        serialized: dict[str, Any],
```

Comment:
Let's replace type hints
dict -> Dict
list -> List

for consistency with the rest of the code base. (dict and list support was added in python 3.10)